subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -68,13 +65,6 @@ class AndroidSantaTrackerSmokeTest extends AbstractSmokeTest {
             "The configuration :detachedConfiguration7 was resolved without accessing the project in a safe manner.  This may happen when a configuration is resolved from a different project.  See https://docs.gradle.org/${GradleVersion.current().version}/userguide/viewing_debugging_dependencies.html#sub:resolving-unsafe-configuration-resolution-errors for more details. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
             "The configuration :detachedConfiguration8 was resolved without accessing the project in a safe manner.  This may happen when a configuration is resolved from a different project.  See https://docs.gradle.org/${GradleVersion.current().version}/userguide/viewing_debugging_dependencies.html#sub:resolving-unsafe-configuration-resolution-errors for more details. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
             "The configuration :detachedConfiguration9 was resolved without accessing the project in a safe manner.  This may happen when a configuration is resolved from a different project.  See https://docs.gradle.org/${GradleVersion.current().version}/userguide/viewing_debugging_dependencies.html#sub:resolving-unsafe-configuration-resolution-errors for more details. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
-            "Type 'DependencyTask': field 'configurations' without corresponding getter has been annotated with @Input. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
-            "Type 'DependencyTask': field 'outputDir' without corresponding getter has been annotated with @OutputDirectory. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
-            "Type 'DependencyTask': field 'outputFile' without corresponding getter has been annotated with @OutputFile. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
-            "Type 'LicensesTask': field 'dependenciesJson' without corresponding getter has been annotated with @InputFile. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.",
```

Comment:
Didn't we remove the `LicenseTask` from the `santa-tracker` build: https://github.com/gradle/santa-tracker-android/commit/d4d6b2470f9752f19198341ccd4e6a9719b82832

That might also explain why the `DependencyTask` doesn't show up any more.

Note that the plugin is not part of AGP, so I think it is fine.

Reply:
I've submitted a PR for the licenses plugin here: https://github.com/google/play-services-plugins/pull/111